src/pages/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+
+import Layout from '../components/Layout'
+import SEO from '../components/Seo'
+import { Box, Button, Grid, Link, Text } from '@theme-ui/components'
+import { Link as RouterLink } from 'gatsby'
+import { APP_MAX_WIDTH } from '../config'
+import purpleBG from '../assets/images/purple-bg.jpg'
+import YouTube from 'react-youtube-embed'
+
+const Home = () => (
+  <Layout>
+    <SEO title="Home" />
+
+    <Box
+      mx="auto"
+      py={4}
+      px={3}
+      sx={{
+        width: '100%',
+        maxWidth: APP_MAX_WIDTH,
+        fontFamily: 'Caviar Dreams',
+        fontSize: 3,
+        fontWeight: 'bold',
+      }}
+    >
+      <Grid columns={[1, 2]} gap={4}>
+        <YouTube id="https://www.youtube.com/embed/I6hmxc4fGXk" />
+        <YouTube id="https://www.youtube.com/embed/0WzacLIkK2c" />
+
+        <Box>
+          {/* TODO: Provide links to these claims */}
+          <Text as="p">
+            Voted{' '}
+            <Link href="https://google.com">
+              &ldquo;Best Live Band in Tennessee&rdquo;
+            </Link>{' '}
+            &{' '}
```

Comment:
Trivial at best, can't you just do `{ ' & ' }` ?

Reply:
Yeah, for sure. That's just WebStorm's automatic handling of spaces. I didn't even look at it.